.github/workflows/label_docs.yml

```
@@ -0,0 +1,32 @@
+name: CI
+
+on:
+  pull_request_target:
+    types:
+      - labeled
+      - unlabeled
+
+jobs:
+  dispatch:
+    runs-on: ubuntu-latest
+    if: github.event.label.name == 'doc needed'
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set workflowId environment variable
+        id: set_workflow_id
+        run: |
+          if [[ "${{ github.event.action }}" == "labeled" ]]; then
+            echo "workflowId=new-migrated-issue.yml" >> $GITHUB_ENV
+          else
+            echo "workflowId=delete-migrated-issue.yml" >> $GITHUB_ENV
+          fi
+
+      - name: Dispatch
+        uses: benc-uk/workflow-dispatch@v1 
+        with: 
+          workflow: ${{ env.workflowId }}
+          repo: noir-lang/docs
+          ref: main 
+          token: ${{ secrets.PAT_TOKEN }} 
```

```suggestion
          token: ${{ secrets.DOCS_REPO_TOKEN }} 
```
I believe we need a token with access to the docs repo added to the secrets

---

Added this to the repo secrets so committing